packages/@okta/vuepress-site/docs/guides/third-party-risk-integration/test-integration/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Test the integration
+---
+
+With the creation of a service application for the third-party risk provider, and the update of the third-party risk provider profile, you can now test the integration using the Risk Events API. In this test, the API sends a sample payload risk signal to the Okta org, which can be consumed by Okta and used to calculate the risk of the authentication.
+
+>**Note**: Only IP related risk signals are available for consumption by the Okta org.
+
+Use the following high-level steps to test the Risk Provider integration:
+
+1. [Create a client assertion for the access token](docs/guides/third-party-risk-integration/create-service-app/#create-a-public-private-key-pair)
+2. [Create an access token](docs/guides/third-party-risk-integration/create-service-app/#create-a-public-private-key-pair)
+3. [Send a risk signal to the Okta org](docs/guides/third-party-risk-integration/create-service-app/#create-a-public-private-key-pair)
+4. [Confirm the response and system log](docs/guides/third-party-risk-integration/test-integration/)
+
+### Create a client assertion
+This procedure creates a signed JSON Web Token (JWT), which is used as the client assertion value required in the request for a scoped access token (the following procedure).
```

Comment:
I don't understand `(the following procedure)`

Reply:
Making an attempt to connect the result of the procedure to the following procedure. But it's not necessary. Naturally flows. I've removed it.